components/page-navigation/src/index.js

```diff
@@ -425,11 +427,29 @@ function mobileView() {
 function insertAfter(referenceNode, newNode) {
   referenceNode.parentNode.insertBefore(newNode, referenceNode.nextSibling);
 }
+
+function insertBefore(el, referenceNode) {
+  referenceNode.parentNode.insertBefore(el, referenceNode);
+}
 const contentnav = document.querySelector('.sidebar-container');
 const h1Header = document.querySelector('h1');
+const contentLayout = document.querySelector('.main-content');
 
 if (mobileView()) {
   if (contentnav && h1Header) {
     insertAfter(h1Header, contentnav);
   }
 }
+
+function changeOrder() {
```

Comment:
I thought I saw @xjensen changing this behavior in another PR because moving the header around was causing problems. Was that something else Jon?

Reply:
That's right, I removed the under-header insertion last sprint.